ThirdPartyAdapters/unity/unity/src/main/java/com/google/ads/mediation/unity/UnityInitializer.java

```diff
@@ -59,6 +78,7 @@ public void initializeUnityAds(Activity activity, String gameId, IUnityAdsInitia
         mediationMetaData.commit();
```

Comment:
Should we make the adapter version to "3.5.0" ?

Reply:
Made this change in the new PR!